hw/ip/lc_ctrl/data/lc_ctrl_testplan.hjson

```diff
@@ -3,30 +3,123 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: "lc_ctrl"
-  // TODO: remove the common testplans if not applicable
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/mem_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
   entries: [
     {
       name: smoke
       desc: '''
-            Smoke test accessing a major datapath within the lc_ctrl.
+            Smoke test accessing lc_ctrl state transition datapath.
 
             **Stimulus**:
-            - TBD
+            - Initialize lc_ctrl by sending pwrmgr req and otp_ctrl valid random data
+            - Request a valid next LC state by writing CSRs: `transition_target`,
+              `transition_token*`, and `transition_cmd`
 
             **Checks**:
-            - TBD
+            - After lc_ctrl initialization finishes, check lc_ctrl broadcast outputs, check
+              the value of lc_state and lc_transition_cnt CSRs, and check device_id and id_state
+              CSRs
+            - During lc_ctrl state transition request, check if `transition_regwen` register is set
```

Comment:
I feel it's better to test this in a separate test as it's very timing sensitive. I have a sequence to test this kind of reg. FYI
https://github.com/lowRISC/opentitan/blob/master/hw/ip/keymgr/dv/env/seq_lib/keymgr_cfgen_vseq.sv

Reply:
Thank you for your suggestion and reference. I will separate it as you recommended.